main.cc

```diff
@@ -2027,6 +2028,13 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 view_backlog_broker.stop().get();
             });
 
+            supervisor::notify("starting group0 state broadcaster");
+            static sharded<service::group0_state_broadcaster> g0_state_broadcaster;
+            g0_state_broadcaster.start(std::ref(db), std::ref(sys_ks.local()), std::ref(gossiper)).get();
```

Comment:
Does this really deserve a separate service? More services mean harder to maintain initialization procedure (which is already hard).

Reply:
moved to the group0_state_machine